project/task1/task1.py

```diff
@@ -0,0 +1,67 @@
+import math
+import csv
+import copy
+from pathlib import Path
+from pointClass import Point
+from detectFixations import detectFixations
+
+base_path = Path(__file__).parent
+file_path = (base_path / '../data/train.csv').resolve()
+
+# Load assigned data for group 7 to variable 'data'
+with file_path.open() as dataFile:
+    rawData = []
+    for row in csv.reader(dataFile, delimiter=','):
+        if (row[0] == 's7') or (row[0] == 's17') or (row[0] == 's27') or (row[0] == 's3') or (row[0] == 's13') or (row[0] == 's23'):
+            rawData.append(row)
+
+
+# Initially process Points            
+pointsData = [] # Raw data without sid & known; x & y stored as Point class
+for row in rawData:
+    k = (len(row)-2)/2  # Number of Points in current processing Row
+    rowPoints = []  
+    for i in range(2, k+2):
+        x=float(row[i])
+        y=float(row[i+k])
```

Comment:
This part looks off, data is `x1, y1, ....xn, yn`

Reply:
I read x1...xn :)). Then this will be simpler.